plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergOrcFileWriter.java

```diff
@@ -240,6 +242,17 @@ private static int getIcebergId(OrcType orcColumn)
             max = max.setScale(((Types.DecimalType) icebergType).scale());
             return Optional.of(new IcebergMinMax(icebergType, min, max));
         }
+        TimestampStatistics timestampStatistics = orcColumnStats.getTimestampStatistics();
+        if (timestampStatistics != null) {
+            Long min = timestampStatistics.getMin();
+            Long max = timestampStatistics.getMax();
+            if (min == null || max == null) {
+                return Optional.empty();
+            }
+            // Since ORC timestamp statistics are truncated to millisecond precision, this can cause some column values to fall outside the stats range.
+            // We are adding appending 999 microseconds to account for the fact that Trino ORC writer rounds timestamps.
```

Comment:
> rounds

rounds or truncates (as per previous sentence)?

let's make it clear which one is meant here